pkg/client/block/client_test.go

```diff
@@ -139,3 +152,18 @@ type testBlockEventValueStruct struct {
 	Block   *types.Block  `json:"block"`
 	BlockID types.BlockID `json:"block_id"`
 }
+
+type testCometClient struct {
```

Comment:
If we use `clitestutil "github.com/cosmos/cosmos-sdk/testutil/cli"` and not create a new interface, we can just use `clitestutil.NewMockCometRPC` directly and avoid the creation of new types.